api/controllers/AuthController.ts

```diff
@@ -52,32 +51,32 @@ export class AuthController {
   }
 
   @Get('/emailVerification/:email')
-  async resendEmailVerification(@Param('email') email: string): Promise<ResendEmailVerificationResponse> {
-    const user = await this.userAuthService.setAccessCode(email.toLowerCase());
+  async resendEmailVerification(@Params() vEmail: ValidEmail): Promise<ResendEmailVerificationResponse> {
+    const user = await this.userAuthService.setAccessCode(vEmail.email.toLowerCase());
     await this.emailService.sendEmailVerification(user.email, user.firstName, user.accessCode);
     return { error: null };
```

Comment:
nit: the parameter name `vEmail` (and the other renamed parameters) seems somewhat cryptic/less readable and can probably be renamed to follow the same pattern as request bodies (i.e. having the same name as its type, so something like `emailParams`).

Reply:
Actually, @sumeet-bansal is there any particular reason we have the body as the same name as its type? Since both are the same, we could just have it be `body` (or just `params` in this case) since the type keeps it clear what its holding. That way the function definition/calls are a bit less verbose

Reply:
Since we switched from `@Param('uuid')` to `@Params()`, that variable no longer represents just the UUID but all the request params, so `@Params() params` would be a more fitting name.

But then the issue becomes, `@Params() params` is still only the validated UUID/email/etc since that's what the class is defined as, and we'd have to create another class like `@Params() params: GetAttendancesForEventQueryParams`.

So if we want to use `@Params()` everywhere, that becomes a lot of work, since we'd have to do that for every controller method. Probably easiest to just stick with `@Param('email') email: Email` for the time being.

Reply:
Yeah, we could rename the variables `body`. I personally prefer using the type to name the variable because `resendEmailVerificationRequest` seems like a better name to me, but `body` works too, as long as we're consistent.